privacy_policy/en/privacy-notice-product-windows.md

```diff
@@ -55,6 +55,20 @@ The check for available notifications is performed upon user request. When it ha
 
 * Application Identifier
 
+### After the program crashes
+
+If AdGuard crashes, it creates an automatic crash report. Upon the next launch AdGuard will ask you to send it to us. In this report, the following information is being sent:
+
+* Stack trace;
+* Memory dump (only if the crash was caused by an error in the CoreLibs);
```

Comment:
This is a minidump, not a full dump

Reply:
Also, you should simply say "only in case of a crash in the native code" instead of mentioning corelibs

Reply:
Also, add this info to the AG for Mac privacy notice